mcts/src/main/java/org/motechproject/nms/mcts/service/impl/MctsWebServiceFacadeImpl.java

```diff
@@ -47,7 +47,7 @@ public ChildrenDataSet getChildrenData(LocalDate from, LocalDate to, URL endpoin
             result = dataService.DS_GetChildData(settingsFacade.getProperty(Constants.MCTS_USER_ID), settingsFacade.getProperty(Constants.MCTS_PASSWORD),
                     from.toString(DATE_FORMAT), to.toString(DATE_FORMAT), stateId.toString());
         } catch (RemoteException e) {
-            throw new MctsWebServiceException(String.format("Cannot read children data from %s state. Remote Server Error", stateId), e);
+            throw new MctsWebServiceException("Remote Server Error. Cannot read Children data. ", e);
```

Comment:
Seems to me adding the space at the end of the exception message to void a `+ " "` in another place in the code is weird...